src/sentry/static/sentry/app/views/eventsV2/table/index.tsx

```diff
@@ -82,6 +83,7 @@ class Table extends React.PureComponent<TableProps, TableState> {
     setError(undefined);
 
     this.setState({isLoading: true, tableFetchID});
+    metric.mark(`discover-events-start-${apiPayload.query}`);
```

Comment:
Not sure if there's a better thing to use here instead of the query, seems like `tableFetchID` is being used to avoid dupe requests, but `metric.mark` expects a string and a string of the symbol is always going to be `"Symbol('tableFetchID')"`

Reply:
That's correct.

It's to ensure stale data doesn't get rendered in the UI.

I think this `metric.mark()` is fine here, just need to add `metric.measure()` for cases when responses are dropped: https://github.com/getsentry/sentry/pull/17558/files#r390594143